vendor/vendor.json

```diff
@@ -15,70 +15,34 @@
 			"revisionTime": "2018-08-30T22:02:26Z"
 		},
 		{
-			"checksumSHA1": "GaJLoEuMGnP5ofXvuweAI4wx06U=",
+			"checksumSHA1": "CGj8VcI/CpzxaNqlqpEVM7qElD4=",
```

Comment:
what is are plan going forward for vendor or modules stuff?

Reply:
probably switch to gomodules in the future, since k8s is moving to that. Will do that in a separate patch.

Reply:
OK. I used modules in the stargazer patches. You can then do a go mod vendor and it will produce a vendor dir also, but the mod lists are used first and the vendor is used as hints. I don't know all the ins and outs yet but it works for the local and docker builds.